src/markdown/partners.md

```diff
@@ -287,6 +212,37 @@ You will probably need to check the following values:
 - **users**: you can user the users hash to pick up informations on the user behind the delegatorId. You will probably pick up their email in order to [generate tokens](#delegation-flows) with the module flow to access the data they shared with you.
 - **organisations**: and finally, you may want to know which organisations a user is part of by looking in the `organisations` hash corresponding to the `organisationId` found at path `users[delegatorId].organisationsIds`.
 
+## Delegation flows
+
+Getting access to device data is done through a delegation mechanism: you have to "impersonate" the users to get access to their data. The prerequisite is that the user must have activated at least one of your modules.
```

Comment:
👍  The "impersonate" aspect is important and was missing in the previous documentation